parser/src/parse.ts

```diff
@@ -205,7 +206,12 @@ export async function parse(rawArgs: string[]) {
     process.exit(1)
   }
 
-  const repoDir = args.path ?? "."
+  const truckroot_or_cwd = args.gittruckroot ?? process.cwd()
```

Comment:
This is meant to allow for running the parser from`git-truck` and `git-truck/parser/` directly

Reply:
Great change! 
Why the rename though? I don't really like the name `truckroot_or_cwd`

Reply:
Can you maybe explain the reasoning for having two args for specifying which directory to analyze?

Reply:
- gittruckroot should always point to the git truck root directory. Only used when calling parser from the root npm script.
- repoDir, is the target path to analyse(just like before)

Reply:
Okay, cool. Makes sense.